config/_default/menus.toml

```diff
@@ -146,71 +152,87 @@
   parent = "nexus"
 
 [[main]]
-  name = "Equity in Open Science"
-  url = "/equityinos"
+  name = "Developing Countries & OS"
+  url = "/os-developing-world"
   weight = 35
   parent = "nexus"
 
 [[main]]
   name = "Educators' Corner"
   url = "/educators-corner"
-  weight = 35
+  weight = 36
   parent = "nexus"
-
+  
 [[main]]
-  name = "Glossary"
-  url = "/glossary"
-  weight = 36
+  name = "Equity in Open Science"
+  url = "/equityinos"
+  weight = 37
   parent = "nexus"
 
 [[main]]
-  name = "Impact of O&R on students"
-  url = "/impact"
-  weight = 37
+  name = "Glossary"
+  url = "/glossary/english"
```

Comment:
Hi @flavioazevedo,

It seems there's a redirection issue with the glossary page. Currently:

- On my localhost setup, clicking the "GLOSSARY" menu item redirects to `/glossary/english` instead of `/glossary`.
- On the live site [https://forrt.org/glossary/](https://forrt.org/glossary/), the redirection is working as expected, directing users to `/glossary`.

However, if this PR is merged, it will cause the redirection to change to `/glossary/english`, which might not be the intended behavior.

Could you please change  this? 

Thanks!

Reply:
This does reflect the issue we are seeing at the test environment at the moment. Thanks for the extra pair of eyes Richard!